packages/context-editor/src/ContextEditor.tsx

```diff
@@ -75,12 +95,28 @@ export default function ContextEditor(props: ContextEditorProps) {
 		return state;
 	});
 
+	// this is slightly evil and should not be taken as an example of good api design
+	// it basically allows you to retrieve the value of the editor from the parent component
+	// whenever you want, rather than having to do it through `onChange` (which would also cause a re-render)
+	// this makes (some) sense in this case bc the editor is almost fully uncontrolled:
+	// you cannot pass in an `EditorState` that holds the actual value
+	useImperativeHandle(
+		props.getterRef,
+		() => ({
+			getCurrentState: () => editorState,
+		}),
+		[editorState]
+	);
+
```

Comment:
this is nifty, but a bit hacky. as i mention in the body, we might want to make the context editor more of a controlled component in the future in order to sync transluded pubfields, eg if i change `abstract:title` in the form i might expect it to change automatically here. we should first figure out transclusion tho